sdk/storage/storage-blob/test/appendblobclient.spec.ts

```diff
@@ -76,6 +76,19 @@ describe("AppendBlobClient", () => {
     assert.equal(downloadResponse.contentLength!, content.length);
   });
 
+  it("appendBlock with progress report", async () => {
+    await appendBlobClient.create();
+
+    const content = "Hello World!";
+    await appendBlobClient.appendBlock(content, content.length, {
+      onProgress: () => {}
```

Comment:
minor: The PR enhanced the coverage, and is a great work! One question is, most of the test use empty callbacks, is that by design? Possibly, we may change at least one of them to be a useful callback, so we can validate the callback itself works.

Reply:
Even the body is empty, the callback is invoked. I've verified that before the fix these test failed.